include/ck/ck.hpp

```diff
@@ -178,6 +178,9 @@
 // flag to enable (1) or disable (0) the debugging output in some kernels
 #define DEBUG_LOG 0
 
+// flag to run kernel warm-up: 0 do not run warm-up; 1 run warm-up
+#define RUN_WARMUP 0
```

Comment:
This will change default behavior or kernel launcher.

```stream_config.time_kernel_ ``` could be to ```false``` to disable warning up and timing

https://github.com/ROCmSoftwarePlatform/composable_kernel/blob/afdfef74f786e6e36fcb99108e306a717c00b79b/include/ck/host_utility/kernel_launch.hpp#L21

it could be set through client API
https://github.com/ROCmSoftwarePlatform/composable_kernel/blob/afdfef74f786e6e36fcb99108e306a717c00b79b/client_example/01_gemm/gemm.cpp#L154

Reply:
@asroy Please review if your request is resolved. The PR is blocked unless you approve it.